### PR TITLE
website_forum: easy inserting pictures

### PR DESCRIPTION
**Pull request by sja-odoo**
Task: https://www.odoo.com/web#id=34691&view_type=form&model=project.task&action=333&active_id=131&menu_id=4720

Pad: https://pad.odoo.com/p/r.eadde644a7e0ea0c88060b9400330750

### CODE REVIEW

File: addons/website_forum/models/forum.py [hunk 1]
```diff
@@ -197,7 +197,7 @@ class Post(models.Model):
 
     name = fields.Char('Title')
     forum_id = fields.Many2one('forum.forum', string='Forum', required=True)
-    content = fields.Html('Content', strip_style=True)
+    content = fields.Html('Content', sanitize=False)
```
Comment (qsm-odoo) [inline addons/website_forum/models/forum.py]: Why would you make that change ? I think post contents should be sanitized ? :smile:

File: addons/website_forum/static/src/css/website_forum.css [hunk 2]
```diff
@@ -183,3 +183,6 @@ button.btn-link.text-muted {
 .o_forum_user_bio .o_forum_tooltip_header {
   min-height: 75px;
 }
+.oe_no_empty {
+  word-wrap: break-word;
+}
```
Comment (qsm-odoo) [inline addons/website_forum/static/src/css/website_forum.css]: Could you explain this to me ?

File: addons/website_forum/models/forum.py (hunk 1 above)
Comment (sja-odoo) [reply addons/website_forum/models/forum.py]: Yes @qsm-odoo  indeed content should be sanitized, but that time it was not allowing to insert iframe tag for posting videos on forum, but now it is fixed in master, so now can remove this

File: addons/website_forum/static/src/css/website_forum.css (hunk 2 above)
Comment (sja-odoo) [reply addons/website_forum/static/src/css/website_forum.css]: @qsm-odoo it was creating problem, see video https://drive.google.com/file/d/0B8SK9KKMbAOjVDhGQ2JJblI1QkU/view
 but this also has been fixed so we can remove this too :p

**Comment by danger89** [comment]
So should we just close this pr?

**Comment by mart-e** [comment]
Already fixed in master

**Pull request closed without merging**